docs/building_fledge/building_fledge.rst

```diff
@@ -561,10 +561,10 @@ In order to use the latest version for Fledge, add the following lines at the en
   export LD_LIBRARY_PATH=/usr/local/lib64
 
 
-Installing PostgreSQL 9.6
+Installing PostgreSQL 13 
```

Comment:
Please remove the version from the heading so that it is generic.

Reply:
fixed